packages/bpk-component-phone-input/src/BpkPhoneInput.js

```diff
@@ -97,6 +101,17 @@ const BpkPhoneInput = (props: Props) => {
     disabled: !!disabled,
   };
 
+  let phoneDisplayValue;
+
+  if (countryCodeMask) {
```

Comment:
Can this code be run through prettier? Or set to be consisten of brace style rather than a mix of brace and brace less

Reply:
addressed